docs/tutorials/using_apis/flow_control.md

```diff
@@ -30,20 +32,20 @@ Here is an example that doesn't include sleeping:
 
 ### Thread
 
-Threads are the most efficient way to blink an LED. During the waiting period, it is possible to take advantage of Mbed OS optimizations to automatically conserve power and deal with other tasks. While this is not the most visually appealing method, nor the simplest, it is the preferred way for large scale deployments:
+Threads are the most efficient way to blink an LED. During the waiting period, it is possible to take advantage of Mbed OS optimizations to automatically conserve power and deal with other tasks. While this is not the most visually appealing method, nor the simplest, it is the preferred way for large scale deployments:<!--why?-->
```

Comment:
I think because it's the most efficient in the first sentence?

Reply:
So why not small deployments? If it's most efficient, when will we not use it?